### ISSUE TITLE
Invalid parameter passed to CRT function! 

### ISSUE THREAD

**Issue by DwordXor** [x64dbg/x64dbg#2094]
Hello

I use the last snapshot and xLCB plugin from ThunderCls.
If i try to exports comments have i this error.

**Comment by mrexodia** [comment]
Does it work on any executable? Do you have very long comments perhaps?

Anyway, you can export the database from the File menu directly...

**Comment by DwordXor** [comment]
What mean you with this link ,the bug "Invalid parameter passed to CRT function! "
come from this plugin?

**Comment by mrexodia** [comment]
Yes, it’s an issue that happens when you install a plugin, so it should be
dealt with there.
